doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -111,15 +107,14 @@ Persistent Mounts
 To mount CephFS in your file systems table as a kernel driver, add the
 following to ``/etc/fstab``::
 
-    [{ipaddress}:{port}]:/ {mount}/{mountpoint} ceph [name=username,secret=secretkey|secretfile=/path/to/secretfile],[{mount.options}]
+    {name}@{fs_name}=/ {mount}/{mountpoint} ceph [mon_host={ipaddress},secret=secretkey|secretfile=/path/to/secretfile],[{mount.options}]
 
 For example::
 
-    :/     /mnt/ceph    ceph    name=admin,noatime,_netdev    0       2
+    foo@cephfs=/     /mnt/ceph    ceph    mon_host=192.168.0.1:6789,noatime,_netdev    0       2
 
-The default for the ``name=`` parameter is ``guest``. If the ``secret`` or
-``secretfile`` options are not specified then the mount helper will attempt to
-find a secret for the given ``name`` in one of the configured keyrings.
+If the ``secret`` or ``secretfile`` options are not specified then the mount helper
+will attempt to find a secret for the given ``name`` in one of the configured keyrings.
```

Comment:
You dropped the part about the default for the name parameter being "guest". Does this mean that leaving off the username will now imply that it shouldn't use cephx? That seems fine if so, but we should definitely make it clear in the doc.

Reply:
Well, the user name (ceph user) has been made mandatory with this change even if the cephx is disabled for a cluster.

Reply:
(which is kind of equivalent to how it works now -- default user "guest" is used with or without cephx)

Reply:
It might be good to allow people to leave out both the cephuser and fsname fields, so you could mount using a minimal syntax like:

```@=/```

Then, the mount helper or the kernel could fill in those fields with appropriate defaults. Alternately, leaving out the ceph username could be shorthand to not use cephx at all (unauthenticated connection).

Reply:
Actually, I think requiring the name and fsname is probably the right thing to do. Let's just call this one resolved.